src/analysis/locate.ml

```diff
@@ -862,7 +906,16 @@ let get_doc ~config ~env ~local_defs ~comments ~pos =
         `Found ({ Location. loc_start=pos; loc_end=pos ; loc_ghost=true }, None)
       | Some ctxt ->
         log ~title:"get_doc" "looking for the doc of '%s'" path ;
-        from_longident ~config ~pos ~env ~lazy_trie ctxt `MLI lid
+        begin match from_longident ~config ~pos ~env ~lazy_trie ctxt `MLI lid
+        with
+        | `At_origin ->
+          `Found ({ Location. loc_start=pos; loc_end=pos ; loc_ghost=true }, None)
```

Comment:
If we are at origin — use the current location (same behaviour as `None` is returned from `inspect_context` before.